pkg/apimachinery/utils/meta.go

```diff
@@ -34,8 +34,9 @@ const AnnoKeyRepoPath = "grafana.app/repoPath"
 const AnnoKeyRepoHash = "grafana.app/repoHash"
 const AnnoKeyRepoTimestamp = "grafana.app/repoTimestamp"
 
+// LabelKeyDeprecatedInternalID gives the deprecated internal ID of a resource
 // Deprecated: will be removed in grafana 13
-const labelKeyDeprecatedInternalID = "grafana.app/deprecatedInternalID"
+const LabelKeyDeprecatedInternalID = "grafana.app/deprecatedInternalID"
```

Comment:
do we also need org ID for this to work with on-prem instances that have more than one org?

Reply:
As far as I'm aware, internal IDs are unique across a grafana instance, it is solely uids that are only unique per org. However, we will still have this on a namespaced object, so regardless we should be safe.